sector-builder-ffi/src/api.rs

```diff
@@ -164,6 +164,37 @@ pub unsafe extern "C" fn sector_builder_ffi_get_seal_status(
     raw_ptr(response)
 }
 
+/// Returns the health of all sealed sectors.
+#[no_mangle]
+pub unsafe extern "C" fn sector_builder_ffi_get_all_sealed_sector_status(
+    ptr: *mut SectorBuilder,
+) -> *mut responses::GetAllSealedSectorStatus {
```

Comment:
BLOCKING: Drop this function and add your `FFISectorStatus` to the `FFISealedSectorMetadata` struct. We already have a `sector_builder_ffi_get_sealed_sectors` method which produces all the information a caller would want about a sealed sector.

If you are concerned about the performance implications of adding the disk reading and checksum-computing behavior to an existing method call, make the `FFISectorStatus` field optional and modify `sector_builder_ffi_get_all_sealed_sector_status` to accept a boolean value `check_status: bool`.